test/_init_tests.js

```diff
@@ -292,7 +290,7 @@ function beforeTest() {
 // Global cleanup of tags added during tests. Cool to add more
 // to selector.
 afterEach(function() {
-  consoleSandbox.restore();
+  consoleSandbox.verify();
```

Comment:
Please wrap this in a try block and in the catch augment the error message with instructions for how to add an expectation.